client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -188,17 +188,11 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	handleLaunchSite = task => () => {
+		const { siteId } = this.props;
 		if ( task.isCompleted ) {
```

Comment:
If you click "Launch site" after skipping it and then re-expanding the task then `task.isCompleted` is true and nothing happens.

Reply:
Thanks!  I missed this bit during the refactor.  Should be fixed up now.